models/room.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const RoomSchema = new Schema({
+	room_id: {
+		type: String,
+		required: true
+	},
+	addedTracks: [
+		{
+			info: {
```

Comment:
Can I ask what info will be for my own understanding?

Reply:
info will be "Track Title - Artist"

Reply:
trying to match the data being stored in Play Queue